# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/llvm/hotfix-m68k.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- llvm-16.0.1.src/CMakeLists.txt.vanilla	2023-05-01 13:53:50.367554486 +0200
+++ llvm-16.0.1.src/CMakeLists.txt	2023-05-01 13:54:07.362555451 +0200
@@ -432,6 +432,7 @@
   Hexagon
   Lanai
   LoongArch
+  M68k
   Mips
   MSP430
   NVPTX
